tests/test_bucketlist_api.py

```diff
@@ -0,0 +1,125 @@
+        response = self.client.post('/auth/login',
+                                    data=correct_credentials,
+                                    content_type='application/json')
+
+        content = json.loads(response.get_data())
+        self.access_token = content['access_token']
+
+    def test_it_creates_bucketlist(self):
+        new_bucketlist = json.dumps({"list_title": "Third List", "list_description": "This is the decription"})
+
+        response = self.client.post('/bucketlists',
+                                    data=new_bucketlist,
+                                    content_type='application/json',
+                                    headers={'Authorization': 'JWT %s' % self.access_token})
+
+        content = json.loads(response.get_data(as_text=True))
+        self.list_id = content['bucketlist']['list_id']
+        self.assertEqual(response.status_code, 201)
+
+    def test_it_updates_bucketlist(self):
+        new_bucketlist = json.dumps({"list_title": "Third List", "list_description": "This is the decription"})
+
+        response = self.client.post('/bucketlists',
+                                    data=new_bucketlist,
+                                    content_type='application/json',
+                                    headers={'Authorization': 'JWT %s' % self.access_token})
+
+        content = json.loads(response.get_data(as_text=True))
+        self.list_id = content['bucketlist']['list_id']
+
+
+        new_bucketlist = json.dumps({"list_title": "Third List Updated", "list_description": "This is the updated decription"})
+
+        response = self.client.put('/bucketlists/%s' % self.list_id,
+                                    data=new_bucketlist,
+                                    content_type='application/json',
+                                    headers={'Authorization': 'JWT %s' % self.access_token})
+
+        content = json.loads(response.get_data(as_text=True))
+        self.assertEqual(response.status_code, 200)
+
+        self.assertEqual('The bucketlist with ID %s was updated' % self.list_id,
+                         content['message'])
+
+    def test_it_deletes_bucketlist(self):
+        new_bucketlist = json.dumps({"list_title": "Another List", "list_description": "This is the decription"})
+
+        response = self.client.post('/bucketlists',
+                                    data=new_bucketlist,
+                                    content_type='application/json',
+                                    headers={'Authorization': 'JWT %s' % self.access_token})
+
+        content = json.loads(response.get_data(as_text=True))
+        list_id = content['bucketlist']['list_id']
+
+        response = self.client.delete('/bucketlists/%s' % list_id,
+                                      headers={'Authorization': 'JWT %s' % self.access_token})
+        content = json.loads(response.get_data(as_text=True))
+
+        self.assertEqual('The bucketlist with ID %s was deleted' % list_id,
```

Comment:
Double check to ensure that it is deleted. You could do a get with the id of the deleted bucketlist and check for 404 and error message